### PR TITLE
Fix: Remove Citizen. to bring format up-to-date

### PR DESCRIPTION
**Pull request by alberttheprince**
Remove unnecessary Citizen. to bring things up to date with current practices in the FiveM community.

Replaced:

Citizen.CreateThread
Citizen. Wait
Citizen.Trace

### CODE REVIEW

File: dlc_finance/office1.lua [hunk 1]
```diff
@@ -116,8 +116,8 @@ FinanceOffice1 = {
             elseif doorSide:lower() == "right" then
                 FinanceOffice1.Safe.isRightDoorOpen = true
             else
-                Citizen.Trace("[bob74_ipl] Warning: " .. doorSide .. " is not a correct value. Valid values are:")
-                Citizen.Trace("left right")
+                print("[bob74_ipl] Warning: " .. doorSide .. " is not a correct value. Valid values are:")
+                print("left right")
```
Comment (TheIndra55) [inline dlc_finance/office1.lua:120]: Replacing it here has a different effect since `Trace` doesn't add a newline at the end.

**Comment by Legacy-TacticalGamingInteractive** [comment]
What does this benefit compared to the old way? Curious for other older scripts we run if its worth doing the same.

**Comment by alberttheprince** [comment]
> What does this benefit compared to the old way? Curious for other older scripts we run if its worth doing the same.

It's mostly a "best practice" type change. Maybe some insanely minor performance improvement no one could detect.

I think https://docs.fivem.net/docs/scripting-reference/runtimes/lua/functions/Citizen.Await is the only thing that needs Citizen.

File: dlc_finance/office1.lua (hunk 1 above)
Comment (alberttheprince) [reply dlc_finance/office1.lua:120]: Combined:

                print("[bob74_ipl] Warning: " .. doorSide .. " is not a correct value. Valid values are: left right")

**Review by TheIndra55 — APPROVED** [review]

**Comment by TheIndra55** [comment]
Thanks for your contribution :)

**Pull request merged**